### PR TITLE
feat: allow configuring a multiScmName for the kokoro job

### PR DESCRIPTION
**Pull request by chingor13**
This allows you to specify a `multiScmName` in the `release-trigger.yml` config file. If set, we will trigger the Kokoro job as a `multi_scm` job and provide the git SHA as the commit for the specified `multi_scm_name`.

### CODE REVIEW

File: packages/release-trigger/src/bot.ts [hunk 1]
```diff
@@ -64,7 +64,8 @@ async function doTriggerWithLock(
   pullRequest: PullRequest,
   token: string,
   logger: GCFLogger,
-  installationId: number
+  installationId: number,
+  multiScmName?: string
```
Comment (SurferJeffAtGoogle) [inline packages/release-trigger/src/bot.ts:68]: Add @params to the function's doc comment for installationId and multiScmName.

File: packages/release-trigger/src/bot.ts [hunk 2]
```diff
@@ -112,7 +120,8 @@ async function doTrigger(
   pullRequest: PullRequest,
   token: string,
   logger: GCFLogger,
-  installationId: number
+  installationId: number,
```
Comment (SurferJeffAtGoogle) [inline packages/release-trigger/src/bot.ts:123]: Add @params to the function's doc comment for installationId and multiScmName.

**Review by SurferJeffAtGoogle — CHANGES_REQUESTED** [review]

File: packages/release-trigger/src/bot.ts (hunk 1 above)
Comment (chingor13) [reply packages/release-trigger/src/bot.ts:71]: Done.

File: packages/release-trigger/src/bot.ts (hunk 2 above)
Comment (chingor13) [reply packages/release-trigger/src/bot.ts:129]: Done.

**Review by SurferJeffAtGoogle — APPROVED** [review]

**Pull request merged**